external_worker/local/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  worker:
+    hostname: worker
+    image: enigmampc/worker_sw:0.0.1
```

Comment:
I would set the default as `worker_hw` since this is the intended use of this file.